HACKING.md

```diff
@@ -150,6 +150,28 @@ If you're planning to test changes, it would be best to fork
 this repo so that you do your work there. The workflow
 requires a remote repo to which to push changes.
 
+### Creating AWS credentials config
+
+If you are in production where we upload builds to s3 OR you want to
+test uploading to s3 as part of pipeline development you need to
+create a credentials config as a secret within openshift.
+
+First create a file named `config` (the file must be named `config`) with
```

Comment:
> (the file must be named `config`)

Should be able to just change the invocation below to `--from-file=config=/path/to/config` to make this more foolproof. (Though when I created it I used the older `oc secrets new` interface).